packages/math-input/src/components/input/math-input.tsx

```diff
@@ -908,11 +908,21 @@ class MathInput extends React.Component<Props, State> {
                 overrides.css. */}
                 <div
                     className="keypad-input"
-                    // @ts-expect-error - TS2322 - Type 'string' is not assignable to type 'number | undefined'.
-                    tabIndex={"0"}
+                    tabIndex={0}
```

Comment:
There was no need to have this ts error, and adding the index as a string was actually breaking our ability to focus on the span!